lighthouse-core/config/default_and_dbw.json

```diff
@@ -0,0 +1,412 @@
+{
+  "passes": [{
+    "recordNetwork": true,
+    "recordTrace": true,
+    "gatherers": [
+      "url",
+      "https",
+      "viewport",
+      "theme-color",
+      "manifest",
+      "accessibility",
+      "content-width"
+    ]
+  },
+  {
+    "passName": "offlinePass",
+    "recordNetwork": true,
+    "gatherers": [
+      "service-worker",
+      "offline"
+    ]
+  },
+  {
+    "gatherers": [
+      "http-redirect",
+      "html-without-javascript"
+    ]
+  }, {
+    "recordNetwork": true,
+    "passName": "dbw",
+    "gatherers": [
+      "https",
+      "styles",
+      "url",
+      "./gather/gatherers/dobetterweb/appcache",
```

Comment:
now that you've expanded `getAuditList` and `getGathererList` to include `dobetterweb/`, all of these can drop the path and just start with the directory (e.g. `"dobetterweb/appcache"`)